README.md

```diff
@@ -1,3 +1,326 @@
+    │   │   ├── index.jsx
+    │   │   └── searchbar.playground.jsx
+    │   ├── Avatar
+    │   │   ├── avatar.mdx
+    │   │   ├── avatar.playground.jsx
+    │   │   └── index.jsx
+    │   ├── Badge
+    │   │   ├── badge.mdx
+    │   │   ├── badge.playground.jsx
+    │   │   └── index.jsx
+    │   ├── BottomSheet
+    │   │   ├── bottomsheet.playground.jsx
+    │   │   └── index.jsx
+    │   ├── Button
+    │   │   ├── button.mdx
+    │   │   ├── button.playground.jsx
+    │   │   └── index.jsx
+    │   ├── ButtonGroup
+    │   │   ├── buttongroup.playground.jsx
+    │   │   └── index.jsx
+    │   ├── Card
+    │   │   ├── card.playground.jsx
+    │   │   └── index.jsx
+    │   ├── CheckBox
+    │   │   ├── checkbox.playground.jsx
+    │   │   └── index.jsx
+    │   ├── Divider
+    │   │   ├── divider.playground.jsx
+    │   │   └── index.jsx
+    │   ├── Header
+    │   │   ├── header.playground.jsx
+    │   │   └── index.jsx
+    │   ├── Icon
+    │   │   ├── icon.playground.jsx
+    │   │   └── index.jsx
+    │   ├── Image
+    │   │   ├── index.jsx
+    │   │   └── input.playground.jsx
+    │   ├── Input
+    │   │   ├── index.jsx
+    │   │   └── input.playground.jsx
+    │   ├── ListItem
+    │   │   ├── index.jsx
+    │   │   └── listitem.playground.jsx
+    │   ├── Overlay
+    │   │   ├── index.jsx
+    │   │   └── overlay.playground.jsx
+    │   ├── Pricing
+    │   │   ├── index.jsx
+    │   │   └── pricing.playground.jsx
+    │   ├── Rating
+    │   │   ├── index.jsx
+    │   │   └── searchbar.playground.jsx
+    │   ├── SearchBar
+    │   │   ├── index.jsx
+    │   │   └── searchbar.playground.jsx
+    │   ├── Slider
+    │   │   ├── index.jsx
+    │   │   └── slider.playground.jsx
+    │   ├── SocialIcon
+    │   │   ├── index.jsx
+    │   │   └── socialicon.playground.jsx
+    │   ├── Text
+    │   │   ├── index.jsx
+    │   │   └── tooltip.playground.jsx
+    │   ├── Tile
+    │   │   ├── index.jsx
+    │   │   └── tile.playground.jsx
+    │   ├── ToolTip
+    │   │   ├── index.jsx
+    │   │   └── tooltip.playground.jsx
+    │   └── utils
+    │       └── createReactViewBaseConfig.js
+    ├── index.css
+    ├── index.js
+    ├── logo.svg
+    ├── pages
+    │   ├── explore
+    │   │   └── index.jsx
+    │   └── home
+    │       ├── animation.json
+    │       ├── animation.jsx
+    │       └── index.jsx
+    └── serviceWorker.js
+
+
+```
+
+## Releated Projects
+ <a href="https://reactnativeelements.com/">
+    <img alt="react-native-elements" src="https://user-images.githubusercontent.com/5962998/65694309-a825f000-e043-11e9-8382-db0dba0851e3.png" width="300">
+ </a>
+ <a href="http://rne-storybook.surge.sh/">
+    <img alt="react-native-elements-storybook" src="https://user-images.githubusercontent.com/17120764/56100603-19199580-5f45-11e9-96a6-dd80f61b4932.png" width="450">
+  </a>
+
+## Documentation
+
+[View the full docs of React Native elements  here](https://reactnativeelements.com/docs/overview)
+
+## Contributing
```

Comment:
This seems to be copied from RNE core
Pls make the required changes so that it reflects data from this project